sway-lib-std/src/storage/storage_map.sw

```diff
@@ -68,11 +68,7 @@ impl<K, V> StorageKey<StorageMap<K, V>> where K: Hash {
     /// }
     /// ```
     pub fn get(self, key: K) -> StorageKey<V> where K: Hash {
-        StorageKey {
-            slot: sha256((key, self.field_id)),
-            offset: 0,
-            field_id: sha256((key, self.field_id)),
-        }
```

Comment:
Admittedly, I don't write too much Sway code myself, maybe that's the reason I think the original code looks a bit easier to read

Reply:
The primary purpose for this is encapsulation. Admittedly the changes in the std-lib to use this should be broken up into new lines, not sure why the formatter didn't flag this. The original method of defining each variable is still possible.